system-status.py

```diff
@@ -180,7 +180,7 @@ def add_db(result, cfg):
 
     with psql.cursor() as cur:
         cur.execute(f"SELECT ts,size FROM {tbl} ORDER BY id DESC LIMIT 40")
```

Comment:
Yes, but then it will take the 40 first ones, instead of the 40 last ones. If you want the last 40 ones in the right order, you need to use an offset, which cannot be negative, etc. :) (or maybe they have a new attribute?)
I found it easier and clearer to do it like that (but I forgot the reverse...)